cinema/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)

```

Comment:
for description it would be better to write null=True, blank=True

Reply:
Fixed